### ISSUE TITLE
Make dashboard page a true dashboard for productivity?

### ISSUE THREAD

**Issue by Matthias84** [jonaswinkler/paperless-ng#378]
Hi, first I really like the dashboard / intro page as it's pretty informative for first-start users.

But if you work a lot with the application, I noticed, that it doesn't help me to do work, so I only skip it.
Instead, it could help me as a personal entrypoint, similar like it shows already the installation stats
Some ideas for (often used) functions:
* tagcloud
* saved search views (maybe also as inline list?)
* new in inbox (unprocessed -> need to be checked)
If we allow users to hide the first steps, there might be enough space to show up this personalized content.

**Comment by shamoon** [comment]
You can already put saved searches on the dashboard, and / or the sidebar. Once you add to dashboard the instructions go away. Then you can make as many dashboard views as you want. I have an Inbox one and a Recently Added one.

tag cloud is an idea but curious how that would be useful?

**Comment by Matthias84** [comment]
Thanks for the tip, I just remembered the sidbar option.
But wouldn't it be useful to indicate this feature also at the dashboard page itself? Something like an add button, or a close x button on the introduction widget?

**Comment by shamoon** [comment]
https://github.com/jonaswinkler/paperless-ng/blob/04a0d01b21a98abb8ee058ce2556a4af52e909e8/src-ui/src/app/components/dashboard/widgets/welcome-widget/welcome-widget.component.html#L7

?

**Comment by Matthias84** [comment]
Depends on you, I missed this hint and I guess UI patterns are more intuitive than long hints?

**Comment by jonaswinkler** [comment]
* Closing the "First steps" widget won't make saved views appear on the dashboard, so there's no need for that.
* I think that the hint is concise enough, and there's even a screenshot about where to click.
* Eventually I'd like to make the dashboard more customizable, but this is what we have right now, and I believe it's working out okay for the majority.

**Comment by Matthias84** [comment]
I agree, that this isn't an essential improvement. Maybe my mockup can show some ideas and you can decide if you pick them? :wink: 
![dashboard remix](https://user-images.githubusercontent.com/217511/105215271-5d41bd00-5b51-11eb-842e-bce18b4f0974.png)

Another nice improvement would be to keep the lists layout (list, thumbs, details) when you set them up via the document browser.
I know there are some django apps which provide widget and dashboard functionality, so maybe they offer a good featureset?

**Comment by shamoon** [comment]
Can I ask what browser youre using? Logo shouldnt be so large, that's one thing that needs fixing =)

**Comment by shamoon** [comment]
Actually I think youre on an older version, unless it still looks like that for you

**Comment by jonaswinkler** [comment]
> Actually I think youre on an older version, unless it still looks like that for you

See bottom left.

> As for the other stuff, I think the point @jonaswinkler was making is that maybe for a future release, it does seem like a lot of work for not that much benefit and there are higher priority things at the moment

Exactly!

> Maybe my mockup can show some ideas and you can decide if you pick them? 😉

Yes, this is exactly what I'd like to do at some point.

> I know there are some django apps which provide widget and dashboard functionality, so maybe they offer a good featureset?

The entire front end is written in Angular / Typescript, so Django apps won't help.

**Comment by sebastientaggart** [comment]
FWIW I found this ticket looking to request a more flexible dashboard... I had no idea it was possible to save views!  Very useful.  Personally my vote would be _against_ a tag cloud, I don't find them useful.

**Comment by epheterson** [comment]
I'm in the same boat as @sebastientaggart, I came here hoping to make the dashboard page useful. It'd be nice if it were more obvious that saving a view would make that page functional. I don't think it's mentioned in the dashboard help text?

There should at least be a little tip at the bottom like:
Tip: Save a view and choose 'Show on Dashboard' to have this first-time user text replaced with a document list.
